solid-acp.ttl

```diff
@@ -0,0 +1,281 @@
+prefix acp: <http://www.w3.org/ns/solid/acp#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix dc: <http://purl.org/dc/terms/>
+prefix vann: <http://purl.org/vocab/vann/>
+
+<http://www.w3.org/ns/solid/acp#>
+    a owl:Ontology ;
+    rdfs:label "Access Control Policy Language (ACP)"@en ;
+    dc:description "The Access Control Policy Language (ACP) is a language for describing, controlling, and granting access to resources."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:seeAlso <https://solid.github.io/authorization-panel/acp-specification/> ;
+    cito:citesAsAuthority
+      <https://www.w3.org/TR/rdf11-concepts/>,
+      <https://www.rfc-editor.org/info/rfc2119>,
+      <https://www.rfc-editor.org/info/rfc8174>,
+      <https://www.rfc-editor.org/info/rfc8288>,
+      <https://www.w3.org/TR/vc-data-model/> ;
+    dc:issued "2022-05-18"^^xsd:date ;
+    vann:preferredNamespacePrefix "vann" ;
```

Comment:
I think this should be `vann:preferredNamespacePrefix "acp"`, right?

Reply:
Good catch, absolutely it should.

Reply:
```suggestion
    vann:preferredNamespacePrefix "acp" ;
```